src/visu.f90

```diff
@@ -320,7 +322,8 @@ subroutine end_snapshot(itime, num)
         write(fmt3,'("(A,F16.4)")')
         write(fmt4,'("(A,F16.12)")')
 
-        open(newunit=is,file="./data/snap"//trim(num)//".ini",action='write',status='replace')
+        write(filename, '(A, I0, A)') "./data/snap", num, ".ini"
```

Comment:
This statement allocates the character array "filename" ?

Reply:
I need to double check this one - it didn't cause any issues though